docs/home/installation.md

```diff
@@ -248,6 +248,38 @@ podAnnotations: {}
 
 > **Tip**: List all releases using `helm list`.
 
+# Trivy Operator Installation
```

Comment:
I would add the Docker Extension as a tutorial and then link here to it since it does not seem too straightforward and is more about the use rather than the installation. What do you think?

Reply:
actually made the link to Trivy Operator part of the Helm installation. That makes more sense to me (didn't change much)

Reply:
>I would add the Docker Extension as a tutorial and then link here to it since it does not seem too straightforward and is more about the use rather than the installation. What do you think?

Hmm. I thought Trivy Operator, Docker Extensions, and other tools would move to `Integrations` or `Tools`. Do we have links in the `Installation` section to all the tools?